apps/etterlatte-saksbehandling-ui/client/src/components/person/behandlingsliste.tsx

```diff
@@ -65,34 +50,50 @@ const lenkeTilBehandling = (behandlingSammendrag: IBehandlingsammendrag): string
 }
 
 export const Behandlingsliste = ({ behandlinger }: { behandlinger: IBehandlingsammendrag[] }) => {
+  const sorterteBehandlinger = behandlinger.sort((a, b) =>
+    new Date(b.behandlingOpprettet!) > new Date(a.behandlingOpprettet!) ? 1 : -1
+  )
+
+  /**
+   * TODO:
+   *  Burde ha en form for paginering av behandlinger. Visning av tabellen og henting av vedtakssammendrag henger
+   *  seg opp når det er for mange elementer i listen. Eksempelvis testbruker STOR SNERK som har 210 behandlinger i
+   *  skrivende stund.
+   */
   return (
-    <>
+    <BehandlingPanel>
+      <Heading size={'medium'}>Behandlinger</Heading>
+
       <Table zebraStripes>
         <Table.Header>
           <Table.Row>
-            {kolonner.map((col) => (
-              <Table.HeaderCell key={`header${col}`}>{col}</Table.HeaderCell>
-            ))}
+            <Table.HeaderCell>Reg. dato</Table.HeaderCell>
+            <Table.HeaderCell>Behandlingstype</Table.HeaderCell>
+            <Table.HeaderCell>Utenlandstilsnitt</Table.HeaderCell>
+            <Table.HeaderCell>Årsak</Table.HeaderCell>
+            <Table.HeaderCell>Status</Table.HeaderCell>
+            <Table.HeaderCell>Virkningstidspunkt</Table.HeaderCell>
+            <Table.HeaderCell>Vedtaksdato</Table.HeaderCell>
+            <Table.HeaderCell>Resultat</Table.HeaderCell>
```

Comment:
Lurer litt på om resultatet her må endres litt. Såvidt jeg kan se fra koden så blir ikke lenger vilkårsresultatet oppdatert i behandling. Det betyr at det er vedtaket som er styrende for resultatet (som også gir mening vil jeg tro). Kanskje vedtakssammendraget bør ta med feks vedtakType som gir informasjon om dette er en innvilgelse, opphør, endring osv?

Reply:
Har begynt på dette, men på en annen branch. Ville at denne branchen _kun_ skulle være opprydding uten ny funksjonalitet.